tests/cluster_tests/test_qsubs.sh

```diff
@@ -8,7 +8,7 @@ fails=''
 
 pushd ../../framework
 RAVEN_FRAMEWORK_DIR=$(pwd)
-source ../scripts/establish_conda_env.sh --load
+#source ../scripts/establish_conda_env.sh --load
```

Comment:
Do these commented lines (also the ones below in this file) need to be kept?

Reply:
Removed commented out lines.